src/components/SearchHit.js

```diff
@@ -136,9 +136,6 @@ export class SearchHit extends Component {
                   verticalAlign: 'inherit',
                 }}
               />
-              <span id={canvasLabelHtmlId}>
```

Comment:
I'm surprised you're removing the canvas label here. Was this intentional?

See https://github.com/ProjectMirador/mirador/issues/2683 for what we're expecting the layout to look like.

Reply:
Thanks Chris. I guess I was totally wrong about this, and it is an expected annotation label. I'll just close this PR.
<img width="288" alt="Screenshot 2023-11-22 at 12 57 03 PM" src="https://github.com/ProjectMirador/mirador/assets/1328900/bacbd84f-5a46-442d-9716-eeaea3919b94">